mitosheet/src/mito/Mito.tsx

```diff
@@ -1003,12 +1003,35 @@ export const Mito = (props: MitoProps): JSX.Element => {
         'mito-taskpane-container-narrow': narrowTaskpaneOpen,
     })
 
+    const [resizingTaskpane, setResizingTaskpane] = useState(false);
+
     return (
         <div 
             className="mito-container" 
             data-jp-suppress-context-menu 
             ref={mitoContainerRef} 
             tabIndex={0}
+            onMouseMove={(event) => {
```

Comment:
I think we need to add more listeners here to cancel the drag. E.g. what happens if you:
1. Start resizing
2. Leave the Mito Container
3. Then, mouse up

When you go back inside the Mito Container, you're still resizing. 

I guess this is actually fine -- you just have to click to make it stop.

Reply:
Yeah I think this is ok - fixing it might cause more trouble than good.